src/orca-loader/LoaderPaymentElement.res

```diff
@@ -296,10 +296,11 @@ let make = (componentType, options, setIframeRef, iframeRef, mountPostMessage) =
 
       let oElement = Window.querySelector(selector)
       let classesBase = optionsDict->OrcaUtils.getClasses("base")
-      let additionalIframeStyle = componentType->Utils.isOtherElements ? "height: 2rem;" : ""
+      let additionalIframeStyle =
+        componentType->Utils.isOtherElements ? "height: 2rem;" : "height: 0;"
```

Comment:
Have you checked if this will work with payment element too ?
`componentType->Utils.isOtherElements ? "height: 2rem;" : "height: 0;"`

Reply:
Yes. This will remove the initial height which the iframe has of 150px